qrenderdoc/Code/Interface/PersistantConfig.h

```diff
@@ -458,3 +458,8 @@ storing custom settings to be persisted without needing to modify code.
 
   QString m_Filename;
 };
+
+static inline bool IsHostADB(const QString &hostname)
+{
+  return hostname.startsWith(lit("adb:"));
+}
```

Comment:
This could be a member function of `RemoteHost` since every time we're calling it, we pass in the hostname from there.